app/class/invertedIndexClass.js

```diff
@@ -0,0 +1,195 @@
+/**
```

Comment:
Your file name `InvertedIndexClass`, you don't need to add the `Class` to the file name.